core/code/map_data_render.js

```diff
@@ -1,14 +1,13 @@
 // MAP DATA RENDER ////////////////////////////////////////////////
 // class to handle rendering into leaflet the JSON data from the servers
+/* global L */
 
-
-
-window.Render = function() {
+window.Render = function () {
   this.portalMarkerScale = undefined;
 }
 
 // start a render pass. called as we start to make the batch of data requests to the servers
-window.Render.prototype.startRenderPass = function(level,bounds) {
+window.Render.prototype.startRenderPass = function (bounds) {
```

Comment:
for api compatibility, should this check if the function was given 2 arguments to ignore the first one ?

Reply:
I don't believe this is neccessary. 
I hardly can think of a use-case for plugins here.

If you prefer to support current format keep parameter count instead of adding special cases.
```suggestion
window.Render.prototype.startRenderPass = function (_, bounds) {
```

Reply:
no preference, but can you note this in the PR text so it's not forgotten ?